### PR TITLE
:bug: fix hcl_appscan, handle severity is None #10074

### PR DESCRIPTION
**Pull request by manuel-sommer**
see #10074 

However, the scanfile was not very helpful as a lot of values were rotated (e.g. also severity and cwe) and I had to rotate those back to get something useful.
I figured out that some severity values were None and fixed it with the scan file.

### CODE REVIEW

**Comment by manuel-sommer** [comment]
I have no clue why there is a merge conflict.

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

**Review by hblankenship — APPROVED** [review]

File: unittests/tools/test_hcl_appscan_parser.py [hunk 1]
```diff
@@ -37,3 +37,10 @@ def test_issue_9279(self):
         self.assertEqual(findings[5].mitigation, "Remediation: fix_61771\nAdvisory: attReferrerPolicyHeaderExist")
         self.assertEqual(findings[1].description, "Issue-Type:attHttpsToHttp\nThreat-Class: catInformationLeakage\nEntity: 7089695691196187648\nSecurity-Risks: sensitiveNotOverSSL\nCause-Id: sensitiveDataNotSSL\n")
         self.assertEqual(findings[10].cwe, 1275)
+
+    def test_issue_10074(self):
+        my_file_handle = open("unittests/scans/hcl_appscan/issue_10074.xml")
+        parser = HCLAppScanParser()
+        findings = parser.get_findings(my_file_handle, None)
+        my_file_handle.close()
+        self.assertEqual(4, len(findings))
```
Comment (cneill) [inline unittests/tools/test_hcl_appscan_parser.py:46]: Can you add a check here to ensure the severity fix is working as expected as well?

Comment (manuel-sommer) [reply unittests/tools/test_hcl_appscan_parser.py]: Done, please review again @cneill

**Review by cneill — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Pull request merged**